README.md

```diff
@@ -22,7 +34,8 @@ The application is built using Kotlin.
 
 
 ## Changelog
-
+_03 jan 2022_  
+Fully functional app is ready for PR to Acceptance. No logic added to app yet. So not production ready!
```

Comment:
If you like to maintain a changelog, you might be interested in "Keep a Changelog" (https://keepachangelog.com/en/1.0.0/)

Reply:
Will look in to this!